web/sablony/blackarrow/prihlaska.xtpl

```diff
@@ -201,16 +201,16 @@
 </div>
 
 <!-- begin: covidSekce -->
-<form method="post" class="prihlaska" enctype="multipart/form-data">
+<!-- <form method="post" class="prihlaska" enctype="multipart/form-data"> -->
```

Comment:
Normálně bych tohle chtěl taky spíš přes nějaký option (vyřešený v PHP a pak bychom nevolali tuhle část šablony). Ale s tím Covidem je to tak nejisté, že ho ještě někdy budeme potřebovat, že asi takhle OK.